src/Symfony/Component/Validator/Constraints/BicValidator.php

```diff
@@ -81,5 +92,43 @@ public function validate($value, Constraint $constraint)
 
             return;
         }
+
+        // check against an IBAN
+        if ($path = $constraint->ibanPropertyPath) {
+            if (null === $object = $this->context->getObject()) {
+                return;
+            }
+
+            try {
+                $iban = $this->getPropertyAccessor()->getValue($object, $path);
+            } catch (NoSuchPropertyException $e) {
+                throw new ConstraintDefinitionException(sprintf('Invalid property path "%s" provided to "%s" constraint: %s', $path, \get_class($constraint), $e->getMessage()), 0, $e);
+            }
+        } else {
+            $iban = $constraint->iban;
+        }
+        if ($iban) {
+            $ibanCountryCode = substr($iban, 0, 2);
+            if (ctype_alpha($ibanCountryCode)) {
+                if (substr($canonicalize, 4, 2) !== $ibanCountryCode) {
+                    $this->context->buildViolation($constraint->ibanMessage)
+                        ->setParameter('{{ value }}', $this->formatValue($value))
+                        ->setParameter('{{ iban }}', $iban)
+                        ->setCode(Bic::INVALID_IBAN_COUNTRY_CODE_ERROR)
+                        ->addViolation();
+
+                    return;
```

Comment:
not needed

Reply:
@nicolas-grekas I can remove it but I've done like the other violations raised above

Reply:
Please remove it yes, the code elsewhere is unrelated.